src/main/java/it/pagopa/pagopa/apiconfig/controller/HomeController.java

```diff
@@ -42,7 +54,12 @@ public RedirectView home() {
             @ApiResponse(responseCode = "500", description = "Service unavailable", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = ProblemJson.class)))})
 
     @GetMapping("/info")
-    public ResponseEntity<Void> healthCheck() {
-        return ResponseEntity.ok().build();
+    public ResponseEntity<AppInfo> healthCheck() {
```

Comment:
@jacopocarlini why not use [actuators](https://www.baeldung.com/spring-boot-actuators) ?